server/tests/backends/sql_translator_integration_tests/test_sql_athena_translator_steps.py

```diff
@@ -42,11 +43,15 @@ def boto_session() -> Session:
 @pytest.mark.parametrize(
     "case_id, case_spec_file", retrieve_case("sql_translator", "athena_pypika")
 )
-def test_athena_translator_pipeline(boto_session: Session, case_id: str, case_spec_file: str):
+def test_athena_translator_pipeline(
+    boto_session: Session, case_id: str, case_spec_file: str, available_variables: dict
+):
     pipeline_spec = get_spec_from_json_fixture(case_id, case_spec_file)
 
     steps = [{"name": "domain", "domain": "beers_tiny"}] + pipeline_spec["step"]["pipeline"]
-    pipeline = Pipeline(steps=steps)
+    pipeline = PipelineWithVariables(steps=steps).render(
+        available_variables, nosql_apply_parameters_to_query
```

Comment:
I thought `nosql_apply_parameters_to_query` should never be used with SQL engines ?